_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -349,6 +356,35 @@ public static function register_endpoints() {
 		) );
 	}
 
+	public static function get_plans( $request ) {
+		$data      = get_transient( 'jetpack_plans' );
+
+		/**
+		 * Filter to turn off caching of Jetpack plans
+		 *
+		 * @since 5.9.0
+		 *
+		 * @param bool true Whether to cache Jetpack plans locally
+		 */
+		$use_cache = apply_filters( 'cache_jetpack_plans', true );
```

Comment:
I wonder if it would not better to change the filter to start with `jetpack_` instead of `cache_`? It seems more in line with our other filters.